test_Current_jumps.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from Current_jumps import std_1D
+from Current_jumps import get_n_moms_of_moving_array
+import pytest
+
+# list of inputs and expected results for std_1D() function
+def list_to_test_std_1D():
```

Comment:
This function is not a test, hence I would put that in a module called maybe test_utils and then import in this file